.github/workflows/build_linux.yaml

```diff
@@ -0,0 +1,35 @@
+name: CI-Linux
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-linux:
+    name: build on ubuntu
+    runs-on: ubuntu latest
+    strategy:
+      matrix:
+        openssl: [true, false]
+
+    steps:
+      - name: Checkout code
+        uses: action/checkout@v4
+      - name: Setup environment
+        run: |
+          if [ "$OPENSSL" = "true" ]; then
+            sudo apt-get install -y libssl-dev
+          fi
+      - name: Install libuv
+        run: |
+          apt-get install -y wget pkg-config
+          wget -q -O libuv-v1.26.0.tar.gz https://dist.libuv.org/dist/v1.26.0/libuv-v1.26.0.tar.gz
```

Comment:
1.26 is from 2019, and I think we build everything in PT using 1.42 or something like that, so please update

Reply:
Thanks! Yeah - just copying over from circleci to revive the CI at the moment and getting the build going. Will update to latest!